server/src/main/java/org/elasticsearch/common/collect/Iterators.java

```diff
@@ -36,6 +36,8 @@ public T next() {
         };
     }
 
+    @SafeVarargs
+    @SuppressWarnings("varargs")
```

Comment:
I assumed the `@SafeVarargs` annotation _also_ suppressed this compiler warning but I guess not?

Reply:
You'd think so, but no 🤷